bpfman/src/utils.rs

```diff
@@ -38,8 +40,17 @@ pub(crate) fn read<P: AsRef<Path>>(path: P) -> Result<Vec<u8>, BpfmanError> {
     Ok(data)
 }
 
-pub(crate) fn get_ifindex(iface: &str) -> Result<u32, BpfmanError> {
-    match if_nametoindex(iface) {
+pub(crate) fn get_ifindex(iface: &str, netns: Option<String>) -> Result<u32, BpfmanError> {
```

Comment:
I originally tried to move the get_ifindex down into the code that was run after we entered the netns.  However, I ran into problems because there was code that assumed ifindex was already set in the database, so there were problems.  I though it was messier to try to re-arrange the code than to leave the order as is, and just enter the netns to get the ifindex.  Also, the only operations for which we need to be in the netns are when we get the ifindex, and we we do the actual attach, so I thought it would be better to limit the amount of time that the code is in the other netns to just those operations.  I can try to re-work this if you really want, but I'd prefer to leave it as is.